drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/TestToolsViewImpl.java

```diff
@@ -0,0 +1,203 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
```

Comment:
2019

Reply:
@Rikkola 
Actually this is a renaming - it was RightPanelViewImpl previously

Reply:
@gitgabrio 
`git` is considering this as new file and `CheatSheetViewImpl` as a renaming of `RightPanelViewImpl`. Btw fine for me too keep it as is